Automattic/jetpack
# Add Tracks events to connection section on my jetpack

See linked issue

## Proposed changes:

* This PR adds a few events in the connection section on My Jetpack

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


Yes. It does add a few more Tracks events to My Jetpack, in the Connection section

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch (or run it locally)
* Without connecting, visit `/wp-admin/admin.php?page=my-jetpack`
* Open your browser's developer console on the Network tab
* Add `t.gif` to the text filter
* On My Jetpack, scroll down to the connection section. Click on different links in that section and confirm that events are registered for each link (if you are connected, you'll see links to manage and learn more, otherwise you'll see a "Connect" link)


<img width="514" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/e80cf61b-d4ab-42c3-ad43-2233224108d1">
